app/controllers/concerns/previewable.rb

```diff
@@ -0,0 +1,8 @@
+module Previewable
+  extend ActiveSupport::Concern
+
```

Comment:
What about adding a `set_form_preview_path` method here that needs to be overridden? Seeing as it's needed for this concern to be used.

Reply:
The path will fall back to the `markup#preview` action if `@form_preview_path` is not set in the controller

Adding a `before_action :set_form_preview_path` here will not work since we need `@issue/@evidence` etc. to be set before we set the path here. Is this what you meant?